src/pages/permissions/panels/EditIdentitiesForm.tsx

```diff
@@ -36,8 +36,8 @@ const EditIdentitiesForm: FC<Props> = ({
     notify.failure("Loading details failed", error);
   }
 
-  const nonTlsIdentities = identities.filter(
-    (identity) => identity.authentication_method !== "tls",
+  const fineGrainedIdentities = identities.filter(
+    (identity) => identity.type !== "Client certificate (unrestricted)",
```

Comment:
I think we should reuse the `isUnrestricted` function here as well to establish a single source of truth.